packages/flutter/test/material/ink_well_test.dart

```diff
@@ -92,6 +90,68 @@ void main() {
     await tester.pump(const Duration(seconds: 1));
   });
 
+  testWidgets('ink well changes color on hover', (WidgetTester tester) async {
+    await tester.pumpWidget(Material(
+      child: Directionality(
+        textDirection: TextDirection.ltr,
+        child: Center(
+          child: Container(
+            width: 100,
+            height: 100,
+            child: InkWell(
+              hoverColor: const Color(0xff00ff00),
+              splashColor: const Color(0xffff0000),
+              focusColor: const Color(0xff0000ff),
+              highlightColor: const Color(0xf00fffff),
+              onTap: () {},
+              onLongPress: () {},
+              onHover: (bool hover) {}
+            ),
+          ),
+        ),
+      ),
+    ));
+    final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse);
+    await gesture.addPointer();
+    await gesture.moveTo(tester.getCenter(find.byType(Container)));
+    await tester.pumpAndSettle();
+    final RenderObject inkFeatures = tester.allRenderObjects.firstWhere((RenderObject object) => object.runtimeType.toString() == '_RenderInkFeatures');
+    expect(inkFeatures, paints..rect(rect: const Rect.fromLTRB(350.0, 250.0, 450.0, 350.0), color: const Color(0xff00ff00)));
+  });
+  testWidgets('ink response changes color on focus', (WidgetTester tester) async {
```

Comment:
Minor nit: add a blank line above and below this function.

Reply:
Done.